docs/protos.rst

```diff
@@ -0,0 +1,6 @@
+bayesmix/protos
+
+Protos
+======
+.. raw:: html
+    :file: protos.html
```

Comment:
There should definitely be an introduction to the protos page. I can write it if you want

Reply:
Feel free to write something if you have it already in mind!